sdb/main.py

```diff
@@ -16,18 +16,13 @@
 aapl_avg = pd.DataFrame([aapl_dates,
                          np.convolve(aapl, window, 'same')],
                         index=['date', 'adj_close']).T
-"""StocksDashboard().build_dashboard(data1={'AAPL': AAPL, 'GOOG': GOOG,
-                                         'IBM': IBM, 'MSFT': MSFT},
-                                  data2={'AAPL_avg': aapl_avg},
-                                  params={'line_width': 2, 'color': 'pink'},
-                                  params2={'line_width': 1.5})"""
 
 # Multiple formats for each line.
-StocksDashboard().build_dashboard(data1={'AAPL': AAPL, 'GOOG': GOOG,
-                                         'IBM': IBM, 'MSFT': MSFT},
-                                  data2={'AAPL_avg': aapl_avg},
-                                  params={'GOOG': {'line_dash': 'dashed'},
-                                          'AAPL': {'color': 'blue'}},
-                                  params2={'color': 'orange',
-                                           'line_width': 1.5},
-                                  line_width=2)
+StocksDashboard().build_dashboard(
+    data_list=[{'AAPL': AAPL, 'GOOG': GOOG,
```

Comment:
I think this is ok, though maybe overcomplicating the problem. Maybe a dict would be better and we could add that as title for the plot.